src/main/groovy/life/qbic/datamodel/accounting/Quotation.groovy

```diff
@@ -11,7 +13,7 @@ import life.qbic.datamodel.persons.Person
  * @author: Tobias Koch
  *
  */
-class Offer {
+class Quotation {
```

Comment:
Nit: I remember now that @sven1103 and I discussed this and he mentioned that an offer and a quotation are two separate things in terms of when they are created, paid, and so on.

Reply:
Correct, but this will reflect the Quotation right, I don't see why this should be an Offer but have a QuotationId. There we would have confusion in naming again. At least imo.

Reply:
Right the question then is in whether we should have an OfferId instead of a QuotationId 🤷